rag_web_ui.py

```
@@ -0,0 +1,254 @@
+from flask import Flask, render_template, request, jsonify, stream_template
```

Ruff (0.12.2)

`flask.stream_template` imported but unused

Remove unused import: `flask.stream_template`

(F401)